controls/radgridview/managing-data/add-new-entries.md

```diff
@@ -34,13 +34,13 @@ There are two ways to insert a new row with RadGridView - by pressing the __Inse
 	Me.AddingNewRowsGrid.BeginInsert()
 	{{endregion}}
 
-When adding a new item, an new empty row is created, in which the user can input the desired data.
+When a user adds a new item, an empty row is created in which the user can input data.
 
 #### __Figure 1: The newly created row__
 
 ![The newly created row](images/RadGridView_AddingNewItems_1.png)
 
->If the __IsReadOnly__ property of the RadGridView is set to __True__ or the __CanUserInsertRows__ property is set to __False__, no row will be added.
+>If the __IsReadOnly__ property of RadGridView is set to __True__ or the __CanUserInsertRows__ property is set to __False__, no row is added.
 
 The next step in implementing the adding functionality is to attach event handlers to the __AddingNewDataItem__ and the __RowEditEnded__ events.
```

Comment:
I do not believe a separate section is necessary, but I will move this paragraph in the "Modifying New Entries" section.